app/models/spree/app_configuration_decorator.rb

```diff
@@ -4,6 +4,9 @@
   # we can allow to be modified in the UI by adding appropriate form
   # elements to existing or new configuration pages.
 
+  # General Preferences
+  preference :user_guide_link, :string, default: nil
```

Comment:
Should I add a default user guide link (http://www.openfoodnetwork.org/platform/user-guide/ for example). Right now if no link is provided, it just links to the current page.